deploy/templates/azuredeploy.minimum.json

```diff
@@ -643,7 +643,7 @@
                             },
                             {
                                 "key": "PCS_IOTHUB_EVENTHUBCONSUMERGROUP3",
```

Comment:
I cannot find any reference to the following environment variables in our codes:
* `PCS_IOTHUB_EVENTHUBCONSUMERGROUP2`
* `PCS_IOTHUB_EVENTHUBCONSUMERGROUP3`
* `PCS_IOTHUB_EVENTHUBCONSUMERGROUP4`

But instead, I see that our event processors check the following environment variables:
* `PCS_IOTHUB_EVENTHUB_CONSUMER_GROUP_TELEMETRY`
* `PCS_IOTHUB_EVENTHUB_CONSUMER_GROUP_EVENTS`
* `PCS_IOTHUB_EVENTHUB_CONSUMER_GROUP_ONBOARDING`

Should we use them instead of `PCS_IOTHUB_EVENTHUBCONSUMERGROUP*` ones here ?

Reply:
Nice catch.  It always worked because we fall back to the defaults.   I will open a bug to get this fixed.